### ISSUE TITLE
Introduction: what do we have in Boost?

### ISSUE THREAD

**Issue by anarthal** [cppalliance/site-docs#28]
**Which documentation does this apply to?**
User's Guide

I think that the current introduction doesn't transmit the user why she should use Boost. For example:
```
Particular strengths of the Boost libraries include smart pointers, unit tests, lambda functions, function binding, regex, a file system, to name a few.
```

We already have all of this in the standard, so this is somehow contradicting the other arguments.

I'd say the table on library categories belongs to a reference section, rather to an introduction section, since it's ordered alphabetically and doesn't provide a way for the user to explore the libraries inside each category.

**Comment by alandefreitas** [comment]
Yes. If we're listing specific features boost provides, we should definitely skip anything the STL already provides.

The features could also be more generic, including the library categories (networking, databases, etc) instead of individual libraries (HTTP, MySQL, ...) because listing features related to individual libraries involves too much work in identifying which are more important.
